app/Resources/public/less/overlay.less

```diff
@@ -22,12 +22,12 @@
             background: white;
             padding: 1em;
             height: 90%;
-            overflow: scroll;
+            overflow: auto;
```

Comment:
Yes, this just adds a scrollbar if necessary. E.g. the ugly horizontal scrollbar is now hidden.